pallets/stellar-relay/src/lib.rs

```diff
@@ -174,8 +174,10 @@ pub mod pallet {
 			};
 			let organization_satoshipay =
 				OrganizationOf::<T> { name: create_bounded_vec("SatoshiPay"), id: 1.into() };
-			let organization_wirex =
-				OrganizationOf::<T> { name: create_bounded_vec("Wirex"), id: 2.into() };
+			let organization_franklin_templeton = OrganizationOf::<T> {
```

Comment:
Is there a problem that our current definition (the one used on Amplitude at the moment) uses a different order? There it is:

```
[
  {
    id: 0
    name: Stellar Development Foundation
  }
  {
    id: 1
    name: SatoshiPay
  }
  {
    id: 2
    name: Coinqvest
  }
  {
    id: 3
    name: Blockdaemon
  }
  {
    id: 4
    name: LOBSTR
  }
  {
    id: 5
    name: Public Node
  }
  {
    id: 6
    name: Franklin Templeton
  }
]
```

Maybe it's better to keep it consistent at least.

Reply:
Order shouldn't matter but I changed it anyways

Reply:
Thanks!